src/components/molecules/AnnouncementMessage/AnnouncementMessage.tsx

```diff
@@ -14,42 +14,42 @@ import { LinkButton } from "components/atoms/LinkButton";
 import "./AnnouncementMessage.scss";
 
 export interface AnnouncementMessageProps {
-  banner?: Banner;
   isAnnouncementUserView?: boolean;
 }
 
 export const AnnouncementMessage: React.FC<AnnouncementMessageProps> = ({
-  banner,
   isAnnouncementUserView = false,
 }) => {
   const {
-    isShown: isAnnouncementMessageVisible,
     show: showAnnouncementMessage,
     hide: hideAnnouncementMessage,
   } = useShowHide();
 
+  const venueId = useVenueId();
+  const { currentVenue: venue } = useConnectCurrentVenueNG(venueId);
+
+  const { banner } = venue ?? {};
```

Comment:
you can use ALWAYS_EMPTY_OBJECT here instead of `{}`.

Reply:
doesn't work in this case for some reason
![image](https://user-images.githubusercontent.com/56254806/134662879-1d9ece2b-5b5b-49b0-9c19-7878a4b37655.png)